domains/transfer/Transfer.API/API/Shared/Data/ApplicationDbContext.cs

```diff
@@ -12,11 +12,11 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : ba
     {
     }
 
-    public DbSet<TransferAgreement> TransferAgreements { get; set; }
-    public DbSet<TransferAgreementHistoryEntry> TransferAgreementHistoryEntries { get; set; }
-    public DbSet<ConnectionInvitation> ConnectionInvitations { get; set; }
-    public DbSet<Connection> Connections { get; set; }
-    public DbSet<ClaimAutomationArgument> ClaimAutomationArguments { get; set; }
+    public DbSet<TransferAgreement> TransferAgreements { get; set; } = null!;
```

Comment:
Why is this needed?

Reply:
IDE is complaining about Null references, for our dbcontext. This change doesn't "fix" anything, besides removing annoying warnings, that irritate me while i am developing